src/PyHyperScattering/FileIO.py

```diff
@@ -34,7 +34,24 @@ def __init__(self,xr_obj):
     def savePickle(self,filename):
         with open(filename, 'wb') as file:
             pickle.dump(self._obj, file)
-            
+     
+
+    # - This was copied from the Toney group contribution for GIWAXS.
+    def saveZarr(self,  filename: Union[str, pathlib.Path], mode: str = 'w'):
+         """
+         Save the DataArray as a .zarr file in a specific path, with a file name constructed from a prefix and suffix.
+
+         Parameters:
+             da (xr.DataArray): The DataArray to be saved.
+             base_path (Union[str, pathlib.Path]): The base path to save the .zarr file.
+             prefix (str): The prefix to use for the file name.
+             suffix (str): The suffix to use for the file name.
+             mode (str): The mode to use when saving the file. Default is 'w'.
+         """
+         ds = da.to_dataset(name='DA')
+         file_path = pathlib.Path(filename)
+         ds.to_zarr(file_path, mode=mode)
+       
```

Comment:
Where was this function copied from? I'm not sure where it's getting or using the prefix / suffix strings for setting the filename. 

I understand we will want a saveZarr FileIO method for consistency with the other methods, but I think it would be good to modify the parameters/docstring to enable setting a specific filename/save path. It also should handle Datasets in addition to DataArrays, where it just will not need to first convert to a Dataset.